hummingbot/connector/exchange/latoken/latoken_web_assistants_factory.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional
+
+from hummingbot.connector.exchange.latoken.latoken_connections_factory import LatokenConnectionsFactory
+from hummingbot.core.web_assistant.auth import AuthBase
+from hummingbot.core.web_assistant.rest_post_processors import RESTPostProcessorBase
+from hummingbot.core.web_assistant.rest_pre_processors import RESTPreProcessorBase
+from hummingbot.core.web_assistant.web_assistants_factory import WebAssistantsFactory
+from hummingbot.core.web_assistant.ws_post_processors import WSPostProcessorBase
+from hummingbot.core.web_assistant.ws_pre_processors import WSPreProcessorBase
+
+
+class LatokenWebAssistantsFactory(WebAssistantsFactory):
```

Comment:
```suggestion
class WebAssistantsFactory:
    """Creates `RESTAssistant` and `WSAssistant` objects.

    The purpose of the `web_assistant` layer is to abstract away all WebSocket and REST operations from the exchange
    logic. The assistant objects are designed to be injectable with additional logic via the pre- and post-processor
    lists. Consult the documentation of the relevant assistant and/or pre-/post-processor class for
    additional information.

    todo: integrate AsyncThrottler
    """
    def __init__(
        self,
        rest_pre_processors: Optional[List[RESTPreProcessorBase]] = None,
        rest_post_processors: Optional[List[RESTPostProcessorBase]] = None,
        ws_pre_processors: Optional[List[WSPreProcessorBase]] = None,
        ws_post_processors: Optional[List[WSPostProcessorBase]] = None,
        auth: Optional[AuthBase] = None,
    ):
        self._connections_factory = ConnectionsFactory()
        self._rest_pre_processors = rest_pre_processors or []
        self._rest_post_processors = rest_post_processors or []
        self._ws_pre_processors = ws_pre_processors or []
        self._ws_post_processors = ws_post_processors or []
        self._auth = auth
```

I thought it was quite nice how we do it, since we don't need to change anything in hummingbot framework
self._connections_factory is private right, you suggest we should set private modifier like the following?

```suggestion
waf=WebAssistantsFactory()
waf._connection = LatokenConnectionsFactory()
```

Reply:
we removed LatokenWebAssistantsFactory and put WebAssistantsFactory with custom connections factory, however with setting private _connections_factory